internal/encryption/initializers.go

```diff
@@ -0,0 +1,48 @@
+package encryption
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/CubicrootXYZ/matrix-reminder-and-calendar-bot/internal/configuration"
+	"github.com/CubicrootXYZ/matrix-reminder-and-calendar-bot/internal/log"
+	"maunium.net/go/mautrix"
+	"maunium.net/go/mautrix/crypto"
+	"maunium.net/go/mautrix/id"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// GetCryptoStore initializes a sql crypto store
+func GetCryptoStore(db *sql.DB, config *configuration.Matrix) (crypto.Store, error) {
+	account := fmt.Sprintf("%s/%s", config.Username, config.DeviceID)
+
+	// TODO get propper sql support to work?
+	db, err := sql.Open("sqlite3", "olm.db")
```

Comment:
Move the file to a separate folder so the whole dir can be mounted. Long term: find a solution to get this to work with mariadb/mysql.

Reply:
add this to the docker hub description

Reply:
also mention that deleting the olm.db will result in broken e2ee chats.